single-page.bs

```diff
@@ -374,6 +374,15 @@ urlPrefix: https://www.w3.org/TR/CSP3/#; type: dfn; spec: CSP
     text: frame-ancestors
     text: sandbox
 
+<!-- ********************************** RESOURCE-HINTS ********************************************* -->
+
+urlPrefix: https://www.w3.org/TR/resource-hints/#; spec: RESOURCE-HINTS;
+    type: dfn
```

Comment:
`type: dfn` can be on the line above (semi-colon separated). No reason to pull it down yet.